lib/utils/RequestHandler.js

```diff
@@ -59,11 +59,20 @@ class RequestHandler {
             throw new RuntimeError(101)
         }
 
+        // Build query string based on queryParams in defaultOptions
+        var queryString = ''
+        if (this.defaultOptions.queryParams !== undefined) {
+            var opts = this.defaultOptions
+            queryString = '?' + Object.keys(this.defaultOptions.queryParams).map(function (key) {
+                return key + '=' + encodeURIComponent(opts.queryParams[key])
```

Comment:
actually there is no need to parse of the object, you can just apply the object via `qs` param